db/schema.rb

```diff
@@ -10,12 +10,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_23_153710) do
+ActiveRecord::Schema.define(version: 2020_12_23_161937) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
+  create_table "active_storage_attachments", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "record_type", null: false
+    t.bigint "record_id", null: false
+    t.bigint "blob_id", null: false
+    t.datetime "created_at", null: false
+    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
+    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
+  end
+
+  create_table "active_storage_blobs", force: :cascade do |t|
+    t.string "key", null: false
+    t.string "filename", null: false
+    t.string "content_type"
+    t.text "metadata"
+    t.bigint "byte_size", null: false
+    t.string "checksum", null: false
+    t.datetime "created_at", null: false
+    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
+  end
+
```

Comment:
Are we missing an active_storage_variant_records table definition;

I was comparing to [applies implementation in schema.rb](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/db/schema.rb#L19-L44) and can see a third table. 

Found this on a shallow investigation which implies that rails 6+ required [further active storage migrations](https://stackoverflow.com/questions/58373159/unknown-attribute-service-name-for-activestorageblob)

Reply:
As with `service_name` this appears to be a change in Rails 6.1. It isn't mentioned in the change log but I found this in the PR where it was added: https://github.com/rails/rails/pull/37901#issuecomment-640766714